misc/selinux/cfengine-enterprise.te

```diff
@@ -34,6 +34,7 @@ require {
 	type proc_t;
 	type proc_net_t;
 	type proc_xen_t;
+	type proc_security_t;
```

Comment:
IMO, the changelog entry could be more specific / user-oriented. Some loose suggestions;

```
The CFEngine daemons on SELinux systems are now allowed to read security parameters in /proc/system/kernel through the type proc_security_t
```

```
Fixed SELinux errors relating to CFEngine daemons attempting to access `/etc/proc/sys/kernel`
```